src/main/java/com/codeup/sequoiaspringbootblog/services/AdService.java

```diff
@@ -0,0 +1,37 @@
+package com.codeup.sequoiaspringbootblog.services;
+
+import com.codeup.sequoiaspringbootblog.models.Ad;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class AdService {
+    private List<Ad> ads;
+
+    public AdService() {
+        ads = new ArrayList<>();
+        createAds();
```

Comment:
We're missing the initialization of the lists in the Curriculum, I'm gonna add it